src/protocols/secure_channel/CASESession.cpp

```diff
@@ -25,8 +25,8 @@
  */
 #include <protocols/secure_channel/CASESession.h>
 
-#include <inttypes.h>
-#include <string.h>
+#include <cinttypes>
```

Comment:
Again, I understand the desire to clean things up, but it should be in a separate PR.

Reply:
Undone.